webapp/src/errorList.tsx

```diff
@@ -38,19 +38,27 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
             return `${error.messageText}-${error.fileName}-${error.line}-${error.column}`
         }
 
-        const toggleButton = <sui.Button id='toggleErrorList' className={`toggleErrorList collapse-button large`}
```

Comment:
Should be able to drop the #toggleErrorList rule in errorList.less?

Reply:
Yeah it should just removed it. Thanks